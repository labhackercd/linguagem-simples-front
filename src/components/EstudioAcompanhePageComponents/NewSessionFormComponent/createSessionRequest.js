import axiosInstance from '../../../auth/axiosApi.js'
import {API_SESSIONS_URL} from '../../../api_urls'

export default async function createSessionRequest(info) {

  const response = await axiosInstance.post(API_SESSIONS_URL,info);

  if(response.status===201){

    return response.data;
  }

};
