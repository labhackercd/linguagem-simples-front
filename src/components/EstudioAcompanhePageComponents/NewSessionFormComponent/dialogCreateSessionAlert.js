import React, { useEffect } from 'react';
import Dialog from '@material-ui/core/Dialog';
import Typography from '@material-ui/core/Typography'
import DialogContent from '@material-ui/core/DialogContent';
import DialogContentText from '@material-ui/core/DialogContentText';
import CreateSession from './assets/creating_session.svg'
import Box from '@material-ui/core/Box'

export default function CreatingSessionDialog(props) {
  const [open, setOpen] = React.useState(props.open);
  

  useEffect(() => setOpen(props.open), [props.open]);

  return (
    <div>
      <Dialog
        open={open}
        aria-labelledby="alert-dialog-title"
        aria-describedby="alert-dialog-description"
      >

        <DialogContent>
          <Box display="flex" justifyContent="center">
            <img src={CreateSession} alt="Imagem criando sessão"></img>
          </Box>
          
     
            <Box m={1}>
              <Typography align="center">Criando Sessão</Typography>
              <Typography align="justify">Aguarde enquanto a sessão é criada</Typography>
            </Box>

        </DialogContent>
      </Dialog>
    </div>
  );
}