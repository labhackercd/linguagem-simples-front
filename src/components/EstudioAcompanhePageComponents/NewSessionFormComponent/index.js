import React from "react";
import Box from "@material-ui/core/Box";
import Grid  from "@material-ui/core/Grid";
import Typography  from "@material-ui/core/Typography";
import FormGroup from '@material-ui/core/FormGroup';
import Button from '@material-ui/core/Button';

import { makeStyles } from '@material-ui/core/styles';
import TextField from '@material-ui/core/TextField'
import MenuItem from '@material-ui/core/MenuItem';
import SendIcon from '@material-ui/icons/Send';

import Checkbox from '@material-ui/core/Checkbox';
import FormControlLabel from '@material-ui/core/FormControlLabel';

import 'date-fns';
import DateFnsUtils from '@date-io/date-fns';
import { MuiPickersUtilsProvider, KeyboardDatePicker,
} from '@material-ui/pickers';
import ptBrLocale from "date-fns/locale/pt-BR";

const useStyles = makeStyles({
  root: {
    background:'#007E5A',
    border: 0,
    height: '100%',
    width: '100%',
    position: "fixed",
    display: "flex"
  },
  inputBorderColor:{
      "& .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline": {
        borderColor: "green"
      },
  },
  inputTextColor:{
    color:'green',
  }
});

export default function NewSessionFormComponent(){
  const classes = useStyles();

    return (

        <Box>
            <Grid container>
                <Grid item xs={12}>
                    <Box display="flex" justifyContent="flex-start" paddingBottom={3}>
                        <Typography variant="h4" color="textSecondary">Nova sessão</Typography>
                    </Box>
                </Grid>
                <Grid item xs={12}>
                    <Grid container spacing={2}>
                        <Grid item xs={5}>
                            <Box display="block" justifyContent="flex-start" >
                                <div><Typography variant="h6"> Local </Typography></div>
                                <TextField
                                className={classes.inputBorderColor}
                                id="sessionPlace"
                                variant="outlined"
                                size="small"
                                color="primary"
                                value="Plenário"
                                InputProps={{
                                    className: classes.inputTextColor
                                  }}
                                />
                            </Box>
                        </Grid>
                        <Grid item xs={7}>
                            <Box display="block" justifyContent="flex-start" >
                                <div><Typography variant="h6" color="textSecondary"> Comissão </Typography></div>
                                <TextField id="selectComission" value="1" variant="outlined" size="small" fullWidth={true} disabled select>
                                    <MenuItem value="1">Selecione</MenuItem>
                                </TextField>
                            </Box>
    
                        </Grid>
                        <Grid item xs={5}>
                            <Box display="block" justifyContent="flex-start" >
                                <div><Typography variant="h6"> Data </Typography></div>
    
                                    <MuiPickersUtilsProvider utils={DateFnsUtils} locale={ptBrLocale}>
                                    <KeyboardDatePicker
                                        disableToolbar
                                        variant="inline"
                                        format="dd/MM/yyyy"
                                        value=""
                                        id="sessionDate"
                                        InputLabelProps={{
                                            shrink: true,
                                          }}
                                        inputVariant="outlined"
                                        size="small"
                                        />
                                    </MuiPickersUtilsProvider>

                            </Box>
                        </Grid>
                        <Grid item xs={7}>
                            <Box display="block" justifyContent="flex-start" >
                                <div><Typography variant="h6"> Tipo de Sessão </Typography></div>
                                <TextField id="sessionType" value="1" variant="outlined" size="small" fullWidth={true} select>
                                    <MenuItem value="1">Selecione</MenuItem>
                                </TextField>
                            </Box>
                        </Grid>
                        <Grid item xs={12}>
                            <Box display="block" justifyContent="flex-start" >
                                <div><Typography variant="h6"> Canais de transmissão </Typography></div>
                                <FormGroup aria-label="position" row>
                                    <FormControlLabel
                                    id="acompanheTransmissionChannel"
                                    value="true"
                                    control={
                                        <Checkbox
                                          checked={true}
                                          name="checkedB"
                                          color="primary"
                                        />
                                    }
                                    label="Acompanhe"
                                    className={classes.inputBorderColor}
                                    />
                                    <FormControlLabel
                                    id="twitterTransmissionChannel"
                                    value="Twitter"
                                    control={<Checkbox color="primary" />}
                                    label="Twitter"
                                    />
                                </FormGroup>
                            </Box>
                        </Grid>

                    </Grid>
                </Grid>
            </Grid>
            <Box pt={4} pb={8}>
                <Button
                    variant="contained"
                    color="primary"
                    className={classes.button}
                    startIcon={<SendIcon />}
                >
                    Iniciar Sessão
                </Button>
            </Box>
        </Box>

    );

}

