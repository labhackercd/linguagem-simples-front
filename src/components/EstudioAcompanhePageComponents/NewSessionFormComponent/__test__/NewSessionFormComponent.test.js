import React from 'react'
import {render} from '@testing-library/react';
import ReactDOM from 'react-dom'
import NewSessionFormComponent from './../index'
import { mount} from 'enzyme';



test('Test if NewSessionFormComponent renders without crash', () => {
    const div = document.createElement("div")
    ReactDOM.render(<NewSessionFormComponent></NewSessionFormComponent>, div)
    ReactDOM.unmountComponentAtNode(div)
});


test('Render Local Text', () => {
    const { getByText } = render(<NewSessionFormComponent />);
    const linkElement = getByText('Local');
    expect(linkElement).toBeInTheDocument();
});

test('test onchange of session date input field', () => {
    const  wrapper = mount(<NewSessionFormComponent />);
    const sessionDate = wrapper.find("input").at(2);

    sessionDate.instance().value = "13/20/2012";
    sessionDate.simulate("change");
    expect(wrapper.find("input").at(2).prop('value')).toEqual("13/20/2012");
});

test('test onchange of session type input field', () => {
    const  wrapper = mount(<NewSessionFormComponent />);
    const sessionType = wrapper.find("input").at(3);
    sessionType.instance().value = "presential";
    sessionType.simulate("change");
    expect(wrapper.find("input").at(3).prop('value')).toEqual("presential");
});

test('test onchange of acompanheTransmissionChannel input field', () => {
    const  wrapper = mount(<NewSessionFormComponent />);
    const acompanheTransmissionChannel = wrapper.find("input").at(4);
    acompanheTransmissionChannel.simulate("change",{ target: { checked: false }});

    expect(wrapper.find("input").at(4).prop('checked')).toEqual(false);
});

test('test onchange of twitterTransmissionChannel input field', () => {
    const  wrapper = mount(<NewSessionFormComponent />);
    const twitterTransmissionChannel = wrapper.find("input").at(5);
    twitterTransmissionChannel.simulate("change",{ target: { checked: false }});

    expect(wrapper.find("input").at(5).prop('checked')).toEqual(false);
});

test('test click on submit button', () => {
    const  wrapper = mount(<NewSessionFormComponent />);
    const button = wrapper.find("button").at(0);
    button.simulate('click')

});
