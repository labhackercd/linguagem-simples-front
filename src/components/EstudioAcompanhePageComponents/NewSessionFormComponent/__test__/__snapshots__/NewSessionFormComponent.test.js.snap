// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if NewSessionFormComponent snapshot matches 1`] = `
<div
  className="MuiBox-root MuiBox-root-73"
>
  <div
    className="MuiGrid-root MuiGrid-container"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        className="MuiBox-root MuiBox-root-74"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4 MuiTypography-colorTextSecondary"
        >
          Nova sessão
        </h4>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5"
        >
          <div
            className="MuiBox-root MuiBox-root-75"
          >
            <div>
              <h6
                className="MuiTypography-root MuiTypography-h6"
              >
                 Local 
              </h6>
            </div>
            <div
              className="MuiFormControl-root MuiTextField-root NewSessionFormComponent-inputBorderColor-71"
            >
              <div
                className="MuiInputBase-root MuiOutlinedInput-root NewSessionFormComponent-inputTextColor-72 MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                onClick={[Function]}
              >
                <input
                  aria-invalid={false}
                  autoFocus={false}
                  className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  disabled={false}
                  id="sessionPlace"
                  onAnimationStart={[Function]}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  required={false}
                  type="text"
                  value="Plenário"
                />
                <fieldset
                  aria-hidden={true}
                  className="PrivateNotchedOutline-root-76 MuiOutlinedInput-notchedOutline"
                  style={
                    Object {
                      "paddingLeft": 8,
                    }
                  }
                >
                  <legend
                    className="PrivateNotchedOutline-legend-77"
                    style={
                      Object {
                        "width": 0.01,
                      }
                    }
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7"
        >
          <div
            className="MuiBox-root MuiBox-root-80"
          >
            <div>
              <h6
                className="MuiTypography-root MuiTypography-h6 MuiTypography-colorTextSecondary"
              >
                 Comissão 
              </h6>
            </div>
            <div
              className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
            >
              <div
                className="MuiInputBase-root MuiOutlinedInput-root Mui-disabled Mui-disabled MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                onClick={[Function]}
              >
                <div
                  aria-disabled="true"
                  aria-haspopup="listbox"
                  aria-labelledby="selectComission"
                  className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input Mui-disabled Mui-disabled MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense Mui-disabled"
                  id="selectComission"
                  onBlur={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onMouseDown={null}
                  role="button"
                  tabIndex={null}
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "&#8203;",
                      }
                    }
                  />
                </div>
                <input
                  aria-hidden={true}
                  autoFocus={false}
                  className="MuiSelect-nativeInput"
                  onAnimationStart={[Function]}
                  onChange={[Function]}
                  required={false}
                  tabIndex={-1}
                  value=""
                />
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined Mui-disabled"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
                <fieldset
                  aria-hidden={true}
                  className="PrivateNotchedOutline-root-76 MuiOutlinedInput-notchedOutline"
                  style={
                    Object {
                      "paddingLeft": 8,
                    }
                  }
                >
                  <legend
                    className="PrivateNotchedOutline-legend-77"
                    style={
                      Object {
                        "width": 0.01,
                      }
                    }
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5"
        >
          <div
            className="MuiBox-root MuiBox-root-81"
          >
            <div>
              <h6
                className="MuiTypography-root MuiTypography-h6"
              >
                 Data 
              </h6>
            </div>
            <div
              className="MuiFormControl-root MuiTextField-root"
            >
              <div
                className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                onClick={[Function]}
              >
                <input
                  aria-invalid={false}
                  autoFocus={false}
                  className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  disabled={false}
                  id="sessionDate"
                  onAnimationStart={[Function]}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  required={false}
                  type="text"
                  value="06/08/2020"
                />
                <div
                  className="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-marginDense"
                >
                  <button
                    className="MuiButtonBase-root MuiIconButton-root"
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    type="button"
                  >
                    <span
                      className="MuiIconButton-label"
                    >
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                        />
                        <path
                          d="M0 0h24v24H0z"
                          fill="none"
                        />
                      </svg>
                    </span>
                  </button>
                </div>
                <fieldset
                  aria-hidden={true}
                  className="PrivateNotchedOutline-root-76 MuiOutlinedInput-notchedOutline"
                  style={
                    Object {
                      "paddingLeft": 8,
                    }
                  }
                >
                  <legend
                    className="PrivateNotchedOutline-legend-77"
                    style={
                      Object {
                        "width": 0.01,
                      }
                    }
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </legend>
                </fieldset>
              </div>
              
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7"
        >
          <div
            className="MuiBox-root MuiBox-root-82"
          >
            <div>
              <h6
                className="MuiTypography-root MuiTypography-h6"
              >
                 Tipo de Sessão 
              </h6>
            </div>
            <div
              className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
            >
              <div
                className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                onClick={[Function]}
              >
                <div
                  aria-haspopup="listbox"
                  aria-labelledby="sessionType"
                  className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  id="sessionType"
                  onBlur={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onMouseDown={[Function]}
                  role="button"
                  tabIndex={0}
                >
                  Virtual
                </div>
                <input
                  aria-hidden={true}
                  autoFocus={false}
                  className="MuiSelect-nativeInput"
                  onAnimationStart={[Function]}
                  onChange={[Function]}
                  required={false}
                  tabIndex={-1}
                  value="virtual"
                />
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
                <fieldset
                  aria-hidden={true}
                  className="PrivateNotchedOutline-root-76 MuiOutlinedInput-notchedOutline"
                  style={
                    Object {
                      "paddingLeft": 8,
                    }
                  }
                >
                  <legend
                    className="PrivateNotchedOutline-legend-77"
                    style={
                      Object {
                        "width": 0.01,
                      }
                    }
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            className="MuiBox-root MuiBox-root-83"
          >
            <div>
              <h6
                className="MuiTypography-root MuiTypography-h6"
              >
                 Canais de transmissão 
              </h6>
            </div>
            <div
              aria-label="position"
              className="MuiFormGroup-root MuiFormGroup-row"
            >
              <label
                className="MuiFormControlLabel-root NewSessionFormComponent-inputBorderColor-71"
                id="acompanheTransmissionChannel"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-84 MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-checked-85 Mui-checked MuiIconButton-colorPrimary"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={true}
                      className="PrivateSwitchBase-input-87"
                      data-indeterminate={false}
                      name="checkedAcompanheTransmissionChannel"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Acompanhe
                </span>
              </label>
              <label
                className="MuiFormControlLabel-root"
                id="twitterTransmissionChannel"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-84 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-87"
                      data-indeterminate={false}
                      name="checkedTwitterTransmissionChannel"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Twitter
                </span>
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-88"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <span
          className="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M2.01 21L23 12 2.01 3 2 10l15 2-15 2z"
            />
          </svg>
        </span>
        Iniciar Sessão
      </span>
    </button>
  </div>
</div>
`;
